lib/rules/indent.js

```diff
@@ -1094,6 +1094,19 @@ module.exports = {
                     parameterParens.add(closingParen);
                     addElementListIndent(node.params, openingParen, closingParen, options.FunctionExpression.parameters);
                 }
+
+                if (firstToken.type === "Identifier" && firstToken.value === "async") {
+                    const nextToken = sourceCode.getTokenAfter(firstToken);
```

Comment:
Sorry, missed this the first time, but what happens if the next token is a comment? Could we add a test to ensure that this is behaving as expected?

Reply:
I added the tests.